test/Common/ErrorBoundary.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Button from '../../src/components/Common/Button';
+import ErrorBoundary from '../../src/components/Common/ErrorBoundary';
+
+describe('Check Component creation', () => {
+    const element = shallow(<ErrorBoundary><Button label="Search"/></ErrorBoundary>);

```

Comment:
A good practice is to not rely on objects or variables created at the beginning of the spec and change it on each test, a better approach is to create this object/variable/component on the test level, if you are repeating the same setup every test,  you can use a beforeEach or use a function that will return the object you need.